lib/db/init.ts

```diff
@@ -0,0 +1,38 @@
+import {db} from "../../lib/db";
+
+export function initDB() {
+    const inELW: number = 309;
+    const inECS: number = 529;
+
+    // clear the database
+
```

Comment:
If you want to nuke the database in its entirety you can just delete the DB file or `DROP TABLE lockers` etc.

Reply:
If I do it that way, sqlite3 db.schema < schema.sql will have to be called right?

Reply:
Yes but you can just execute that from Node as well. 
https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md#new-databasepath-options
> Creates a new database connection. If the database file does not exist, it is created. This happens synchronously, which means you can start executing queries right away. You can create an [in-memory database](https://www.sqlite.org/inmemorydb.html) by passing ":memory:" as the first argument. You can create a temporary database by passing an empty string (or by omitting all arguments).

Reply:
`import sql from './schema.sql'` should work (maybe, I haven't tested this)